content/en/serverless/agent_configuration/_index.md

```diff
@@ -0,0 +1,138 @@
+| `DD_APM_IGNORE_RESOURCES` | An exclusion list of regular expressions can be provided to disable certain traces based on their resource name all entries must be surrounded by double quotes and separated by commas.                                              |
+| `DD_APM_LOG_FILE`         | The full path to the file where APM-agent logs are written.                                                                                                                                                                           |
+| `DD_APM_LOG_THROTTLING`   | Limits the total number of warnings and errors to 10 for every 10 second interval.                                                                                                                                                    |
+| `DD_APM_CONNECTION_LIMIT` | The APM [connection limit][4] for the Agent.                                                                                                                                                                                          |
+
+### Advanced networking configuration
+
+| Env Variable             | Description                                                                                                                                                                                                                                                                                                                        |
+|--------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `DD_SKIP_SSL_VALIDATION` | Setting this option to "true" tells the Agent to skip validation of SSL/TLS certificates. Defaults: false                                                                                                                                                                                                                          |
+| `DD_SSLKEYLOGFILE`       | Specifies a destination for TLS master secrets in NSS key log format to allow external programs such as Wireshark to decrypt TLS connections. For more details, see https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS/Key_Log_Format. Use of sslkeylogfile compromises security and should only be used for debugging. |
+| `DD_MIN_TLS_VERSION`     | This option defines the minimum TLS version that will be used when submitting data to the Datadog intake specified in "site" or "dd_url". This parameter defaults to "tlsv1.2". Possible values are: tlsv1.0, tlsv1.1, tlsv1.2, tlsv1.3; values are case-insensitive. Default: "tlsv1.0"                                           |
```

Comment:
Wait, we default to accepting an insecure TLS version?

Reply:
It seems a typo in the comment, there is some redundancy there
https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml#L67
in reality it's v2
https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config.go#L389

thanks for catching it!